packages/gatsby-theme-store/src/components/Error/ErrorHandler.tsx

```diff
@@ -15,10 +15,17 @@ interface Props {
 }
 
 const getReloads = () =>
-  Number(window.sessionStorage.getItem('store:reloads') ?? '0')
+  Number(
+    navigator.cookieEnabled
+      ? window.sessionStorage.getItem('store:reloads') ?? '0'
+      : '0'
```

Comment:
I think this should be big, otherwise we risk entering infinite loop
```suggestion
      : '2'
```